Source/Rive/Private/RiveModule.cpp

```diff
@@ -18,11 +18,16 @@ THIRD_PARTY_INCLUDES_END
 
 void FRiveModule::StartupModule()
 {
+    // for some reason the shader file path is not loading correctly. 
+    FString PluginShaderDir = FPaths::Combine(IPluginManager::Get().FindPlugin(TEXT("Rive"))->GetBaseDir(), TEXT("Shaders"));
+    AddShaderSourceDirectoryMapping(TEXT("/Plugin/Rive"), PluginShaderDir);
```

Comment:
Should this be TEXT("/Plugins/Rive") (plural Plugins)

Reply:
I don't think so since its working.